fetch.bs

```diff
@@ -2368,6 +2368,9 @@ manually. [[!HTML]]
  <li><p>Return <var>potentialDestination</var>.
 </ol>
 
+<p>The <dfn lt="document Accept header value">Document `Accept` header value</dfn> is
+`<code>text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8</code>`.
```

Comment:
Maybe place this next to https://fetch.spec.whatwg.org/#default-user-agent-value? And do we need the `lt` attribute here?